src/prompt/tests.py

```diff
@@ -1 +1,164 @@
-# Create your tests here.
+from django.test import Client
+from django.urls import reverse
+from rest_framework.test import APITestCase
+import json
+from prompt.models import Prompt
+from api.tests import PearTestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+
+class PromptTestCase(APITestCase):
+
+    def setUp(self):
+        pear_test_case = PearTestCase()
+        pear_test_case.setUp()
+        self.user_token = pear_test_case._create_user_and_login()
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.user_token)
+
+    def populate(self):
+        """populates the database"""
+        url = reverse("populate")
+        return self.client.post(url, {"filenames": ["pear_prompts.txt"]}, format='json')
+
+    def test_populate_prompts(self):
+        """tests that the code populates the 27 prompts in pear_prompts.txt"""
+
+        response = self.populate()
```

Comment:
I see you use self.populate() a lot; why don't you put this in the setUp function which runs before each test?